nightly/nightly.txt

```diff
@@ -18,6 +18,7 @@ pytest --timeout=900 sanity/state_sync2.py
 pytest --timeout=900 sanity/state_sync3.py
 pytest --timeout=600 sanity/state_sync_routed.py manytx 100
 pytest --timeout=300 sanity/state_sync_late.py notx
+pytest --timeout=900 sanity/state_sync_massive.py
```

Comment:
If it doesn't work yet, I think there is no point adding it to nightly.

Reply:
Disagree, I think the policy should be that all the tests that should be passing in a fully functional node must be in nightly.